server/api/store.go

```diff
@@ -56,9 +56,12 @@ type storeInfo struct {
 	Status *storeStatus `json:"status"`
 }
 
-const downStateName = "Down"
+const (
+	disconnectedName = "Disconnected"
+	downStateName    = "Down"
+)
 
-func newStoreInfo(store *core.StoreInfo) *storeInfo {
+func newStoreInfo(store *core.StoreInfo, maxStoreDownTime time.Duration) *storeInfo {
```

Comment:
Since we pass the `maxStoreDownTime` to `storeInfo` now, we can provide a `IsDown` method for convinience.

Reply:
This struct just for the api show. no other place use it. I think it is ok.

Reply:
I see.